tests/integration/container/tests/basic_connectivity.test.ts

```diff
@@ -74,6 +74,7 @@ describe("basic_connectivity", () => {
         password: env.databaseInfo.password,
         port: env.databaseInfo.clusterEndpointPort,
         plugins: "failover,efm2",
+        wrapperConnectTimeout: 120000,
```

Comment:
Just curious, is this the only test in this suite that needs the timeout?

Reply:
I've only ever seen that specific test fail but I can add the change to others